src/Amido.Stacks.Data.Documents.CosmosDB.Tests/Integration/IntegrationFixture.cs

```diff
@@ -33,8 +33,9 @@ public async Task InitializeAsync()
                 if (Settings.DatabaseAccountUri.Contains("localhost", StringComparison.InvariantCultureIgnoreCase) &&
                     Environment.GetEnvironmentVariable("VisualStudioEdition") != null)
                 {
+                    var primaryString = this.Settings.DatabasePrimaryKey;
                     Environment.SetEnvironmentVariable(Settings.SecurityKeySecret.Identifier,
-                        "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==");
+                        primaryString);
```

Comment:
Is `primaryKey` a better variable name than `...String`?

Reply:
Updated per feedback.